API/api/tests/test_event.py

```
@@ -0,0 +1,110 @@
+from api.tests.base_TestClass import BaseAPITestCase
+from api.models import Event
+
+
+class CreateEventTests(BaseAPITestCase):
+    def setUp(self):
+        super().setUp()
+        try:
+            from django.urls import reverse
+            self.create_event_url = reverse("create_event")
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)